lib/deprecation-helpers.spec.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { assert } = require('chai');
```

Comment:
Could you switch to the `expect` form of `chai`? I think we're using that consistently.

Ooh, we should add that to Danger too.